src/app/modules/training/components/training/training.component.ts

```diff
@@ -54,7 +54,8 @@ export class TrainingComponent implements OnInit {
   program?: ProgramDtoApi;
   programRun?: ProgramRunApi;
   score = 0;
-  remainingQuestions: QuestionDtoApi[] = [];
+
+  remainingQuestions?: QuestionDtoApi[] = [];
```

Comment:
Tu peux virer le `?`

Reply:
j'ai une erreur ligne 133 si je le retire :
"Type 'undefined' is not assignable to type 'QuestionDtoApi[]'."

Reply:
Ajoute `?? []` ligne 135

Reply:
done 👍

Reply:
C'est plutôt pour l côté pratique de pas avoir à tester la nullité à chaque fois.
Certaines variables ont besoin de faire la différence entre zéro et null (pour les placeholders par exemple) mais pas ici.